tests/php/test-class-wc-connect-api-client.php

```diff
@@ -1,37 +1,58 @@
 <?php
 
+/**
```

Comment:
A lot of the changes here are due to running `phpcs` to lint the file. The lint fix was added as a separated commit here: https://github.com/Automattic/woocommerce-services/pull/2216/commits/3af85dbb64a6511c4fa469107553c1a5f7931c56